javascript/javascript1/week1/startup-Name.js

```diff
@@ -0,0 +1,9 @@
+const firstWords= ["Easy", "Awesome", "Corporate"];
+const secondWords=["Pease","Lemone","Squeezy"];
+let startupName="";
+
+let randomNumber = Math.floor(Math.random() * 2);
+
+
+startupName=firstWords[randomNumber].concat(secondWords[randomNumber]);
```

Comment:
`+` is enough to concatenate two strings.
- You can create two randomNumber variables to get two random indexes, but it is fine. You don't have to change it